src/jest-dom/index.ts

```diff
@@ -1,6 +1,4 @@
 import { addToElementCache, serialize } from '../serialize';
-// eslint-disable-next-line @cloudfour/node/no-extraneous-import
-import { equals as jestEquals } from 'expect/build/jasmineUtils';
```

Comment:
Jest 27 changes the `expect` package to have an export map, so it no longer lets you deep-import a package like this